app/controllers/api/v0/webhooks_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V0
+    class WebhooksController < ApiController
+      respond_to :json
+
+      before_action :authenticate!
+      # before_action :cors_preflight_check
```

Comment:
Not sure if we need these callbacks for the webhooks API

Reply:
yeah we may not need it